app/src/main/java/com/infomaniak/drive/data/services/UploadWorker.kt

```diff
@@ -433,7 +426,7 @@ class UploadWorker(appContext: Context, params: WorkerParameters) : CoroutineWor
             level = SentryLevel.INFO
         })
 
-        if (UploadFile.canUpload(uri, fileModifiedAt, realm) && fileSize > 0) {
+        if (UploadFile.canUpload(uri, fileModifiedAt, realm) && fileSize >= 0) {
```

Comment:
We avoid synchronizing empty files, but we don't know what impact this will have on certain smartphones.